### PR TITLE
MAYA-113309 MayaUSD : Python bindings for prim translators

### PR DESCRIPTION
**Pull request by boudrey**
Prim Reader and Writer python bindings + Sphere test
Skeleton python binding for Shader and Chaser + test

### CODE REVIEW

**Comment by kxl-adsk** [comment]
FYI. @dgovil, @ysiewappl, and @fabal

**Review by ppt-adsk — APPROVED** [review]
Code review done interactively with Yves, only minor points for which Yves has taken notes, mostly comments to be added for better understanding of design and implementation choices.

**Review by JGamache-autodesk — APPROVED** [review]

File: lib/mayaUsd/python/wrapPrimReader.cpp [hunk 1]
```diff
@@ -0,0 +1,329 @@
...
+            context, prim);
+    }
+
+    static void Register(boost::python::object cl, const TfToken& usdInfoId)
+    {
+        UsdMaya_RegistryHelper::g_pythonRegistry = true;
+        UsdMayaShaderReaderRegistry::Register(
+            usdInfoId,
+            [=](const UsdMayaJobImportArgs& args) {
+                return UsdMayaShaderReader::ContextSupport(0);
```
[the remark this answers is not in the document]
Comment (boudrey) [reply lib/mayaUsd/python/wrapPrimReader.cpp:196]: I'm still working on ShaderReader and ShaderWriter, this function will call CanImport in Python and return the proper value.

**Review by kxl-adsk — APPROVED** [review]
With the last commit, you addressed my only concern about _SetUsdPrim being public. One great improvement would be to prevent this method from being used outside of python `__init__`, but it's not a blocker for this pull request.

**Pull request merged**